src/DefaultEnvironment.php

```diff
@@ -263,6 +271,24 @@ public function getBaseConfigRegistry()
         return $this->baseConfigRegistry;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function setFlatConfigRegistry($flatConfigRegistry): EnvironmentFlatConfigRegistryInterface
```

Comment:
Missing argument type `?ConfigBaseConfigRegistryInterface $flatConfigRegistry`

Reply:
Can I really change this in a minor release?